packages/gatsby/src/redux/index.ts

```diff
@@ -32,26 +39,30 @@ const readState = () => {
   } catch (e) {
     // ignore errors.
   }
-  return {}
+  // BUG: Would this not cause downstream bugs? seems likely. Why wouldn't we just
```

Comment:
This is what somebody else should check into. I don't know :/

Reply:
Yeah. Me either, but it's existing functionality.. I need to explore that more with somebody